--- old
+++ new
@@ -1,1 +1,1 @@
-Pojazd trakcyjny – pojazd kolejowy z napędem własnym[1], przystosowany do ciągnięcia lub pchania wagonów, lub posiadający wydzieloną przestrzeń do przewozu osób albo ładunków[2].
+Pojazd trakcyjny – [pojazd kolejowy](./pojazd_kolejowy) z napędem własnym[1], przystosowany do ciągnięcia lub pchania wagonów, lub posiadający wydzieloną przestrzeń do przewozu osób albo ładunków[2].
@@ -12,0 +12,2 @@
+
+